Team/teammembers/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+class TeamMember(models.Model):
+	firstName = models.CharField(max_length=200, null=False, blank=False)
```

Comment:
if all fields are null = True then I can create users 100s of users with all fields null, how will you differentiate?